webserver/views/api/v1/core.py

```diff
@@ -274,13 +312,18 @@ def get_many_highlevel():
 
        {"mbid1": {"offset1": {document},
                   "offset2": {document}},
-        "mbid2": {"offset1": {document}}
+        "mbid2": {"offset1": {document}},
+        "mbid_mapping": {"MBID1": "mbid1"}
        }
 
     MBIDs and offset keys are returned as strings (as per JSON encoding rules).
     If an offset is not specified in the request for an mbid or is not a valid integer >=0,
     the offset will be 0.
-    MBID keys are always lower-case, even if the provided recording MBIDs are upper-case or mixed case.
+
+    MBID keys are always returned in a normalised form (all lower-case, separated in groups of 8-4-4-4-12 characters),
+    even if the provided recording MBIDs are not given in this form. In the case that a MBID is not given in this
```

Comment:
Could clarify that we're talking about MBID given to us in the request (and not the response) in the second sentence.
```suggestion
    even if the provided recording MBIDs are not given in this form. In the case that a MBID in the request is not in this
```